connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/errors/BadRequestException.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package org.apache.kafka.connect.runtime.rest.errors;
+
+import javax.ws.rs.core.Response;
+
+public class BadRequestException extends ConnectRestException {
+
+    public BadRequestException(String message) {
+        super(Response.Status.BAD_REQUEST, message);
```

Comment:
Sigh, perhaps. This possibly falls into the same territory as the metrics stuff where adding these codes probably shouldn't have too high overhead.

I'm fine leaving it without for now since I don't think we're doing this at all atm. We might want a subsequent follow up that ensures you never generate an error without a more specific error code. That could cover defining the different error codes.